chia/server/server.py

```diff
@@ -714,14 +714,11 @@ def get_connections(self, node_type: Optional[NodeType] = None) -> List[WSChiaCo
         return result
 
     async def close_all_connections(self) -> None:
-        keys = [a for a, b in self.all_connections.items()]
-        for node_id in keys:
-            try:
-                if node_id in self.all_connections:
-                    connection = self.all_connections[node_id]
-                    await connection.close()
-            except Exception as e:
-                self.log.error(f"Exception while closing connection {e}")
+        try:
+            for connection in self.all_connections.copy().values():
+                await connection.close()
+        except Exception as e:
+            self.log.error(f"Exception while closing connection {e}")
```

Comment:
one change you introduce here is that if a call to `close()` throws, you abort, previously we would keep trying to close the remaining connections

Reply:
Right, thanks, changed!